pages/article_list_page.rb

```diff
@@ -4,11 +4,10 @@ class ArticleListPage < DemoAppPage
   validate :title, /\ADemo web application - Listing Articles\z/
 
   element :new_article_button, :xpath, "//a[@href='/articles/new']"
-  element :article_button, :xpath, ->(title) { "//a[contains(.,'#{title}')]" }
-  element :destroy_button, :xpath,
-          ->(title) { "//strong[.='#{title}']/following-sibling::a[normalize-space(.)='Destroy'][1]" }
-  element :edit_button, :xpath,
-          ->(title) { "//strong[.='#{title}']/following-sibling::a[normalize-space(.)='Edit'][1]" }
+  element :article_item, :xpath, ->(title) { "//section[@class='article__item'][.//a[.='#{title}']]" }
```

Comment:
2 IMPORTANT RULES!

1. Never use //, only .//
2. Do no use .//xxx[.//yyy], just .//yyy[yyy]

Please review your code again and change everywhere